client/dashboard/app/breadcrumbs/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { Breadcrumbs as BreadcrumbsComponent } from '@automattic/components/src/breadcrumbs';
+import { useMatches } from '@tanstack/react-router';
+import type { BreadcrumbItemProps } from '@automattic/components/src/breadcrumbs/types';
+
+export default function Breadcrumbs() {
+	const matches = useMatches();
+
+	const items = [] as BreadcrumbItemProps[];
+
+	let hasParent = false;
+	matches.forEach( ( match ) => {
+		const breadcrumbItemLabel = match.staticData?.breadcrumbItemLabel;
+		if ( breadcrumbItemLabel || hasParent ) {
+			items.push( {
+				label: breadcrumbItemLabel?.() || '',
```

Comment:
It wouldn't make any sense to have an empty label, so we could move this check before pushing the item.

Reply:
This was a "hack" because the individual settings subpage needs to be in the breadcrumb items; even if it's not shown (because it's the last item).

Let me think on how to make it cleaner...